client/src/cbltest/api/syncgateway.py

```diff
@@ -870,3 +945,17 @@ async def close(self) -> None:
         """
         if not self.__admin_session.closed:
             await self.__admin_session.close()
+
+    async def create_public_session(
```

Comment:
This is adding to a bad design where the internal method `_send_request` is being used by tests.  If you need to use a different session, create a private one inside this class and expose API to do the things that you want.